.github/workflows/dependant-repos.yml

```diff
@@ -0,0 +1,24 @@
+name: Check that repositories submoduling us will still build after this PR
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  android-bindings:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Check that android-bindings still builds
+      uses: ./.github/actions/check-dependant-rust-repo
+      with:
+        remoteRepo: mobilecoinofficial/android-bindings
+        submodulePath: mobilecoin
+        id: android-bindings
+        githubToken: ${{ secrets.GITHUB_TOKEN }}
+        buildCmd: make build
```

Comment:
```suggestion
jobs:
  android-bindings:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3

    - name: Check that android-bindings still builds
      uses: ./.github/actions/check-dependant-rust-repo
      with:
        remoteRepo: mobilecoinofficial/android-bindings
        submodulePath: mobilecoin
        id: android-bindings
        githubToken: ${{ secrets.GITHUB_TOKEN }}
        buildCmd: make build
        
 full-service:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3

    - name: Check that android-bindings still builds
      uses: ./.github/actions/check-dependant-rust-repo
      with:
        remoteRepo: mobilecoinofficial/full-service
        submodulePath: mobilecoin
        id: full-service
        githubToken: ${{ secrets.GITHUB_TOKEN }}
        buildCmd: make build
```

Reply:
Could we add full service to this list?

Reply:
Yes, in fact full-service might be better to start with since I think it actually builds. I'll play around with it. The `buildCmd` suggested above won't work for it, probably need `cargo build` and probably IAS_MODE/SGX_MODE vars. I'll try it out, thanks!